bootloader/src/old_machine_bootloader_linux.c

```diff
@@ -187,15 +196,15 @@ int ping_island(int argc, char * argv[])
 
     char* server = argv[server_i];
 
-    struct response resp;
+    struct response resp = {0};
```

Comment:
How does this work?

Reply:
That initializes all fields to byte \0, aka 0.